lib/main.js

```diff
@@ -200,8 +200,12 @@ export default {
 
   async lint(textEditor, options) {
     const filePath = textEditor.getPath();
-    const text = textEditor.getText();
 
+    if (filePath === null || filePath === undefined) {
```

Comment:
I think this only ever returns a path or `undefined`, no reason not to check both though 😉.